Dockerfile-auth

```diff
@@ -7,13 +7,14 @@ VOLUME ["/usr/src/app"]
 
 EXPOSE 11000
 
-COPY . /usr/src/app/
+COPY requirements.txt /usr/src/app/
+COPY fabric_cf /usr/src/app/
 RUN pip3 install --no-cache-dir -r requirements.txt
 RUN mkdir -p "/etc/fabric/message_bus/schema"
 RUN mkdir -p "/etc/fabric/actor/config"
 RUN mkdir -p "/var/log/actor"
 RUN cp /usr/local/lib/python3.9/site-packages/fabric_mb/message_bus/schema/*.avsc /etc/fabric/message_bus/schema
-RUN pip3 install fabric-am-handlers==1.0rc1
+RUN pip3 install fabric-am-handlers==1.0rc2
```

Comment:
No need to do it now, for the future probably can make the version a variable up top?

Reply:
Saw the update. Looks good